bin/add-roles.sh

```diff
@@ -18,6 +18,7 @@
 ./bin/utils/ccd-add-role.sh "caseworker-ras-validation"
 ./bin/utils/ccd-add-role.sh "admin-access"
 ./bin/utils/ccd-add-role.sh "full-access"
+./bin/utils/ccd-add-role.sh "hearing-schedule-access"
```

Comment:
My comment disappeared, how odd. I think this line is in there already at line 24.